schema/deploy/shipping/views.sql

```diff
@@ -164,4 +164,80 @@ create or replace view shipping.return_results_v1 as
 comment on view shipping.return_results_v1 is
     'View of barcodes and presence/absence results for return of results on website';
 
+
+create or replace view shipping.fhir_encounter_details_v1 as
+
+    with
+        fhir as (
+          select encounter_id,
+                 array_agg(distinct condition."id") as symptoms,
+                 jsonb_object_agg(response."linkId", response.answer) as responses
```

Comment:
This aggregate and others below may want an explicit ordering applied for consistency, particularly since subsequent answers for the same `linkId` will overwrite previous ones. The docs for [_Aggregate Expressions_](https://www.postgresql.org/docs/10/sql-expressions.html#SYNTAX-AGGREGATES) include examples of aggregate ordering.